mkdocs.yml

```
@@ -85,13 +85,44 @@ nav:
         - './microservices/support/scheduler/Ch-Scheduler.md'
         - './microservices/support/eKuiper/Ch-eKuiper.md'
       - Devices Services:
+        - Supported Device Services:
+          - './microservices/device/Ch-DeviceServiceList.md'
```

I like to see the source folders mirror the renders doc structure.
```suggestion
          - './microservices/device/supported/Ch-DeviceServiceList.md'
```